src/api/deploy-test-suite/helpers/generate-test-run-message.js

```diff
@@ -50,7 +50,10 @@ const generateTestRunMessage = (imageName, environment, runId, user) => {
       browser: 'chrome',
       version: 'latest'
     },
-    deployed_by: user,
+    deployed_by: {
+      userId: user.id,
```

Comment:
If this was related to the validation, there is an alternative validation to use id rather than userId